.github/workflows/release.yaml

```diff
@@ -17,6 +17,9 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+        with:
+          # prometheus_interface library is a submodule
+          submodules: recursive
```

Comment:
Thanks for spotting this!

(non-blocking) I believe `true` is enough, just like what we used before https://github.com/canonical/bootstack-actions/blob/main/.github/workflows/_charm-release.yaml#L35